api_tests/providers/preprints/views/test_preprint_provider_request_list.py

```diff
@@ -44,12 +44,14 @@ def test_list(self, app, admin, moderator, write_contrib, noncontrib, pre_mod_pr
         assert res.json['data'][0]['embeds']['target']['data']['id'] == post_mod_preprint._id
 
         # test_filter
-        res = app.get(f'{self.url(pre_mod_provider)}?filter[target]={pre_mod_preprint._id}', auth=moderator.auth)
+        pre_mod_preprint_id = pre_mod_preprint._id.split('_v')[0] if '_v' in pre_mod_preprint._id else pre_mod_preprint._id
+        res = app.get(f'{self.url(pre_mod_provider)}?filter[target]={pre_mod_preprint_id}', auth=moderator.auth)
         assert res.status_code == 200
         assert len(res.json['data']) == 1
         assert res.json['data'][0]['id'] == pre_request._id
 
-        res = app.get(f'{self.url(post_mod_provider)}?filter[target]={post_mod_preprint._id}', auth=moderator.auth)
+        post_mod_preprint_id = post_mod_preprint._id.split('_v')[0] if '_v' in post_mod_preprint._id else post_mod_preprint._id
+        res = app.get(f'{self.url(post_mod_provider)}?filter[target]={post_mod_preprint_id}', auth=moderator.auth)
```

Comment:
Same issue here, switching to base guide and dropping version is not the fix. Maybe there is a bug that is revealed by this code.

Please take a look at if f`v2/providers/preprints/<provider_id>/withdraw_requests/` handles versioned id correctly when it is passed in as the filter: `?filter[target]=<preprint_id>`.